aospy/test/test_automate.py

```diff
@@ -351,3 +354,24 @@ def test_permute_aux_specs(self, calc_suite, calcsuite_specs):
         assert len(actual) == len(expected)
         for act in actual:
             assert act in expected
+
+
+@pytest.mark.parametrize(
```

Comment:
Minor style note: splitting across lines like this, with an open parentheses/bracket/curly bracket and then an immediate line break, is usually only needed when either

1. The items/expression within the overall parentheses/bracket/curly bracket is long enough that this helps it fit onto fewer lines, or at least look nicer.
2. The items aren't themselves too long to fit on one line each, but there are a lot of them, so it makes it easier to read.

In this case, you don't need to split lines at all; it would fit into a single line:

```python
@pytest.mark.parametrize('var', [precip_largescale, condensation_rain])
```

Also note more meaningless parentheses; please omit.